src/web100srv.c

```diff
@@ -1721,39 +1637,32 @@ void NDT_server_main_loop(SSL_CTX *ssl_context, int listenfd) {
         continue;
       }
 
-      /* if ((mchild->next == NULL) && (mchild->running == 0))
-       *   mchild = tmp_ptr;
-       * if (mchild != head_ptr) 
-       */
       tmp_ptr = mchild;
 
       // Update queued clients, send message to client when it moves
       //  up in the queue enough to get closer to running a test.  This
       //  happens when the client falls into the next lower maxqueue bin
-      //  RAC 3/21/10
 
-      int rac;
       if (waiting > (2 * max_clients)) {
+        int wait_data = -1;
+        int num_secs = 0;
```

Comment:
It looks like now all the lines with `45` are being rewritten in this commit so I feel like we should fix them.

```
$ grep -R -n -C 0 "[^0-9a-zA-Z]45[^0-9a-zA-Z]" * --include=*.[ch]
ignore >>> src/test_results_clt.c:49:      printf("45 Mbps T3/DS3 or WiFi 11 a/g  subnet\n");
ignore >>> src/test_results_clt.c:50:      mylinkspeed = 45;
src/web100srv.c:1653:            num_secs = 45;
src/web100srv.c:1942:        new_child->stime = tt + (waiting * 45);
src/web100srv.c:1944:        new_child->stime = tt + ((waiting / max_clients) * 45);
ignore >>> src/heuristics.c:31: *  4:  10 < inter-packet throughput (mbits/second) <= 40 - 45 Mbps T3/DS3 or WiFi 11 a/g subnet
```

Can we `#define WAIT_TIME_SECS_BASE 45` so we don't have these scattered `45`s?